cmd/run/run.go

```diff
@@ -483,6 +485,7 @@ func RunServer(ctx context.Context, config *Config) error {
 		grpc_validator.UnaryServerInterceptor(),
 		otelgrpc.UnaryServerInterceptor(otelgrpc.WithTracerProvider(tp)),
 		middleware.NewRequestIDInterceptor(logger),
+		storeid.NewStoreIDInterceptor(),
```

Comment:
can we follow the format of the existing custom interceptors and add it inside the `middleware` package and not within `middleware/storeid`?

Reply:
https://github.com/openfga/openfga/pull/517#discussion_r1099003416, but I can go either way. Now I think about it I do think it belongs more in `middleware`.

Reply:
Moved to middleware. yay